saleor/webhook/payloads.py

```diff
@@ -208,6 +208,29 @@ def _generate_collection_point_payload(warehouse: "Warehouse"):
     return collection_point_data
 
 
+def _generate_shipping_method_payload(shipping_method, channel):
+    if not shipping_method:
+        return None
+
+    serializer = PayloadSerializer()
+    shipping_method_fields = ("name", "type")
+
+    shipping_method_channel_listing = shipping_method.channel_listings.filter(
+        channel=channel,
+    ).first()
+
+    payload = serializer.serialize(
+        [shipping_method],
+        fields=shipping_method_fields,
+        extra_dict_data={
+            "currency": shipping_method_channel_listing.currency,
+            "price_amount": shipping_method_channel_listing.price_amount,
```

Comment:
We should run `quantize_price` on this value.

Reply:
done